### PR TITLE
Add,view and delete event

### PR DESCRIPTION
**Pull request by Suuuuha**
related to #42

### CODE REVIEW

File: public/map.js [hunk 1]
```diff
@@ -3,13 +3,13 @@ function placeMap () {
   var lat = coords[0];
   var lng = coords[1];
   var mapOptions = {
-    center: new google.maps.LatLng(lat, lng),
+    center: new google.maps.LatLng(lng, lat),
     zoom: 18,
```
Comment (m4v15) [inline public/map.js]: Why? The version on heroku atm looks correct

**Comment by Suuuuha** [comment]
the view (view all events and specific event) and delete are ready to  review @m4v15 @JWLD

File: src/controllers/events/delete_event.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
+const Request = require('request');
```
Comment (jwld) [inline src/controllers/events/delete_event.js:1]: This file is virtually identical to `delete_place` - do you think we should make a generic file for deleting data, or keep them separate @m4v15 ?

File: src/controllers/events/get_all.js [hunk 3]
```diff
@@ -0,0 +1,35 @@
+const Request = require('request');
+const {eventsURL} = require('../../constants/urls.json');
+
+module.exports = (req, res) => {
+  Request(eventsURL, (error, response, body) => {
+    if (error) {
+      return res.render('error');
+    }
+    if (response.statusCode !== 200) {
+      return res.render('error');
```
Comment (jwld) [inline src/controllers/events/get_all.js]: No `statusCode` or `errorMessage` is being passed to these error handlers

File: views/event.hbs [hunk 4]
```diff
@@ -0,0 +1,46 @@
+<ul class="specific-place">
```
Comment (jwld) [inline views/event.hbs:1]: Could you space out the elements in this file a bit? Kinda hard to read. Also quite a few indentation errors yay :v:

File: views/events.hbs [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+  {{#each events}}
+    <li class="place-wrap">
+      <img class="place-wrap__image" src="{{this.imageUrl}}" alt="{{localtext.image}}">
+      <div class="place-wrap__info">
+        <span class="place-wrap__span">{{this.local.name}}</span>
+        {{#if this.local.description}}
+          <span class="place-wrap__span">{{this.local.description}}</span>
+        {{/if}}
+        {{#if this.place}}
+          <span class="place-wrap__span">{{this.place}}</span>
```
Comment (jwld) [inline views/events.hbs]: `this.place` is an object so doesn't render properly

**Comment by m4v15** [comment]
@Suuuuha maybe these should be smaller PRs - like one for view and delete? and then another one for add/edit?

**Comment by Suuuuha** [comment]
@m4v15 yeah I would prefer it as two seperate PRs, I pushed the last requested changes I made. so you can review them and merge once you guys are happy with them.
working on the add/edit on a different PR then!

File: src/controllers/events/delete_event.js (hunk 2 above)
Comment (m4v15) [reply src/controllers/events/delete_event.js:1]: Yeah and if so then maybe extract the place/event/product from the url?

Comment (m4v15) [reply src/controllers/events/delete_event.js:1]: I mean theoretically a lot of these pages will be pretty similar, could make the get_all page generic at some point, but that would require a bit more of a rethink @JWLD

**Review by m4v15 — CHANGES_REQUESTED** [review]

**Comment by m4v15** [comment]
@Suuuuha yeah definitely do as small as PRs as possible smaller ones are much more manageable

Also, I'm gonna be busy this afternoon but will review and merge this evening

File: src/controllers/events/delete_event.js (hunk 2 above)
Comment (jwld) [reply src/controllers/events/delete_event.js:1]: I think we'll leave it separate for now and refactor later

**Review by jwld — APPROVED** [review]

File: public/map.js (hunk 1 above)
Comment (m4v15) [reply public/map.js]: @Suuuuha can you answer this question..?

File: views/event.hbs [hunk 6]
```diff
@@ -0,0 +1,56 @@
...
+  <div class="place_wrap_buttons">
+    <a href="/">{{localText.homePage}}</a>
+    <a href="/{{lang}}/events">{{localText.backList}}</a>
+  </div>
+  <img src="{{event.imageUrl}}" alt="localText.image"></img>
```
Comment (m4v15) [inline views/event.hbs]: `localText.image` should be `{{localText.image}}`

**Review by m4v15 — CHANGES_REQUESTED** [review]

**Comment by Suuuuha** [comment]
@m4v15 I think that change on the map.js was by mistake when we were trying try something..i'll change it back!

**Comment by m4v15** [comment]
@Suuuuha and to think I was so happy when I found out you used `git add . -p`...

**Pull request merged**